packages/server/src/__tests__/plugin/subscriptionCallback/index.test.ts

```diff
@@ -1147,12 +1149,12 @@ describe('SubscriptionCallbackPlugin', () => {
         "ERROR: SubscriptionManager[1234-cats]: Generator threw an error, terminating subscription: The subscription generator didn't catch this!",
         "SubscriptionManager[1234-cats]: Sending \`complete\` request to router with errors",
         "SubscriptionManager: Sending \`check\` request to http://mock-router-url.com for ID: 1234-cats",
-        "SubscriptionCallback: Server is shutting down. Cleaning up outstanding subscriptions and heartbeat intervals",
+        "SubscriptionManager: Heartbeat received response for ID: 1234-cats",
```

Comment:
OK great. Before your change, these three lines had completely disappeared. Your change just makes them come at a slightly different time, and I'm happy to trust you that the reordering is valid for what we're testing.

Your change makes all the tests in this file pass, but there were other failing tests on `version-5` (I probably should have branched off of `version-5` for the Nock upgrade, oops) so instead of trying to get CI to pass here I'll cherry-pick your fix onto `version-5` and close this PR.